docs/other_resources/creating_providers.md

```diff
@@ -104,6 +104,38 @@ func (p *Provider) BooleanEvaluation(
 
 Follow the guidance [here](./caching.md) to implement caching.
 
+## Provider lifecycle, initialization and shutdown
+
+With the release of the v0.6.0 spec, OpenFeature now outlines a lifecycle for provider initialization and shutdown.
+
+flagd providers should do the following to make use of OpenFeature v0.6.0 features:
+
+- start in a `NOT_READY` state
```

Comment:
nit - capitalize the first letter of each sub-point

Reply:
I generally only do this if each sub-point is a sentence, but I constructed this to be sentence fragments.